nebula_decoders/src/nebula_decoders_velodyne/decoders/vls128_decoder.cpp

```diff
@@ -116,16 +116,17 @@ void Vls128Decoder::reset_overflow(double time_stamp)
   }
 
   // Add the overflow buffer points
-  for (size_t i = 0; i < overflow_pc_->points.size(); i++) {
-    auto overflow_point = overflow_pc_->points[i];
+  while (overflow_pc_->points.size() > 0) {
+    auto overflow_point = overflow_pc_->points.back();
```

Comment:
Why did you change the for loop to a while loop? Looks fine, but I'm just curious about the motivation to make the change.

Reply:
Sorry, forgot to mention it before.
If you look at the way the overflow_pc if constructed, the order of the points is reversed.
In the implementation prior to [9091b2f](https://github.com/tier4/nebula/pull/111/commits/9091b2fe40afbbb4fe8c812490eef4ab1faa102d) the new order was kept and copied into the new pc. That is, the output pc was reversed.

While checking the timestamps of the pc I realized that the timestamps were not in ascending order, which led me to find the error. That is way, by reversing the order again while moving the overflow into the new pc, the "correct" order is preserved

Reply:
Ah, I see now. The previous implementation was using `emplace_back()`. Thanks!